do_freebsd.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -xve
+NPROC=`sysctl -n hw.ncpu`
+
+if [ x"$1"x = x"--deps" ]; then
```

Comment:
@tchaikov yes and no...
I do not like comparing strings to "empty"' values. Sometimes strange things will happen.
And perhaps it is old-school, but I've been taught to wrap compares like this with x""x